src/api/entities/Asset/types.ts

```diff
@@ -170,15 +170,15 @@ export interface BaseHistoricAssetTransaction extends EventIdentifier {
   /**
    * Name of the funding round (if provided while issuing the Asset). This value is present only when the value of `event` is `Issued`
    */
-  fundingRound?: string;
+  fundingRound?: string | undefined;
```

Comment:
(nit) The `?` is redundant now.

```suggestion
  fundingRound: string | undefined;
```